inst/include/population_dynamics/population/population.hpp

```diff
@@ -138,13 +135,16 @@ namespace fims
       unfished_spawning_biomass.resize((nyears + 1));
       spawning_biomass.resize((nyears + 1));
       log_naa.resize(nages);
-      log_Fmort.resize(nfleets * nyears);
       log_M.resize(nyears * nages);
-      log_q.resize(nfleets);
       naa.resize(nages);
-      Fmort.resize(nfleets * nyears);
+
       M.resize(nyears * nages);
-      q.resize(nfleets);
+
+      for (size_t fleet_ = 0; fleet_ < this->nfleets; fleet_++)
+      {
+        this->fleets[fleet_]->Initialize(nyears, nages);   
```

Comment:
Under a scenario where there is one fishing fleet operating on two populations, would this structure initialize the fleet twice, for each population? Tagging @msupernaw

Reply:
I just double checked and fleet is initialized in [information.hpp](https://github.com/NOAA-FIMS/FIMS/blob/6a0854ccd0b8fdf3706afd2d432798a34a16fcec/inst/include/common/information.hpp#L134), so maybe remove here?

Reply:
Added to population code review so we don't lose it